### ISSUE TITLE
NTLM autification failed.

### ISSUE THREAD

**Issue by zapbot** [zaproxy/zaproxy#1342]
```
Hi team,
Thank you for your supporting us.
I described issue as below;

<Overview>
I couldn't start security scan on NTLM authentication base web site on Windows Server

<What steps will reproduce the problem?>
1. Configure Authentication property as  follows
     Authentication Method : HTTP/NTLM Authentication
     Add webserver URL (xxx.yyy.zzz) to "Hostname", specify "443" to Port 
     Not specified any other field
2. Configure Users property as follows;
     UserName : demo1
     Enabled : Yes
     Username : Domain\demo1
     Password : password
3. Configure Forced User as follows;
     Specify : demo1
4. Configure Session management as follows;
     Session Manegement Method : Http Authentication Session Management
5. Start Dynamic Scan
6. Code 401, Reason Unauthorized will be appeared

<What is the expected output? What do you see instead?>
Expected Result : code 200, reason  OK
Actual Result   : code 401, reason Unauthorized

<What version of the product are you using? On what operating system?>
OWASP ZAP Version 2.3.1, OS Windows Server 2003,2008 and 2012, IIS 6.0, 7.0 and 7.5,
Language : ja-jp

<Please provide any additional information below.>
Could you please let us know how to configure correctry for NTLM authentication base
we site?


```

Original issue reported on code.google.com by `iehiro.sayama` on 2014-09-19 07:30:23

<hr>
- _Attachment: [Authentication.JPG](https://storage.googleapis.com/google-code-attachments/zaproxy/issue-1342/comment-0/Authentication.JPG)_
- _Attachment: [Result.JPG](https://storage.googleapis.com/google-code-attachments/zaproxy/issue-1342/comment-0/Result.JPG)_

**Comment by kingthorin** [comment]
@iehiro.sayama did you resolve your issue? 

If no further input is received by 20160901 this issue will be closed.

**Comment by thc202** [comment]
Closing, per changes (#4056 and #4061) released in 2.7.0. Please raise a new issue if the latest version still does not work as expected.
